packages/atlas/src/components/_navigation/SidenavViewer/FollowedChannels.tsx

```diff
@@ -1,22 +1,62 @@
-import React, { useState } from 'react'
+import React from 'react'
 import { CSSTransition } from 'react-transition-group'
 
-import { SvgActionChevronB, SvgActionChevronT } from '@/components/_icons'
+import { useBasicChannel } from '@/api/hooks'
+import { Avatar } from '@/components/Avatar'
+import { SvgActionNewChannel } from '@/components/_icons'
+import { SkeletonLoader } from '@/components/_loaders/SkeletonLoader'
+import { NavItem, NavItemProps } from '@/components/_navigation/NavItem'
+import { absoluteRoutes } from '@/config/routes'
+import { useAsset } from '@/providers/assets'
 import { FollowedChannel } from '@/providers/personalData/types'
 import { transitions } from '@/styles'
+import { SentryLogger } from '@/utils/logs'
 
 import {
-  ChannelsItem,
+  BrowseChannelsIcon,
+  BrowseChannelsText,
+  BrowseChannelsWrapper,
+  ChannelTitle,
   ChannelsList,
   ChannelsTitle,
   ChannelsWrapper,
   FollowedChannelsWrapper,
-  ShowMoreButton,
-  ShowMoreIconWrapper,
-  StyledChannelLink,
 } from './FollowedChannels.styles'
 
-const MAX_CHANNELS = 4
+type ChannelNavItemProps = {
+  id: string
+  onChannelNotFound?: (id: string) => void
+}
+
+export const ChannelNavItem: React.FC<NavItemProps & ChannelNavItemProps> = ({
+  id,
+  to,
+  expanded,
+  itemName,
+  isSecondary,
+  onChannelNotFound,
+  onClick,
+}) => {
+  const { channel } = useBasicChannel(id || '', {
+    skip: !id,
+    onCompleted: (data) => !data && onChannelNotFound?.(id),
+    onError: (error) => SentryLogger.error('Failed to fetch channel', 'ChannelLink', error, { channel: { id } }),
+  })
+  const { url: avatarPhotoUrl } = useAsset(channel?.avatarPhoto)
+
+  return (
+    <NavItem to={to} expanded={expanded} itemName={itemName} onClick={onClick} isSecondary={isSecondary}>
```

Comment:
For some reason, when this is collapsing and the item is loading, the channel ID is displayed:
<img width="420" alt="Screenshot 2022-02-09 at 13 41 21" src="https://user-images.githubusercontent.com/12646744/153203795-fd48fd01-49b8-4063-8441-ef3fd41c7652.png">